src/test/java/com/palantir/docker/compose/TestContainerNames.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.docker.compose;
+
+import static java.util.stream.Collectors.toList;
+
+import com.palantir.docker.compose.connection.ContainerName;
+import com.palantir.docker.compose.connection.ImmutableContainerName;
+import java.util.Arrays;
+import java.util.List;
+
+public class TestContainerNames {
```

Comment:
Actually not entirely sure this class is useful. In other instances we're making our own instances of `ContainerName`